components/open-source.vue

```diff
@@ -0,0 +1,206 @@
+<template>
+  <div class="os">
+    <div class="os__content">
+      <div class="os__title">
+        Open source
+      </div>
+      <div class="os__desc">
+        Hawk is an open-source product. Every part of it is available for investigation, improvement, and&nbsp;discussion.
+      </div>
+      <a
+        class="os__link"
+        :href="reposUrl"
+        target="_blank"
+      >
+        Discover the Code »
+      </a>
+      <div class="os__repo">
+        <a
+          class="os__repo-container"
+          :href="reposUrl"
+          target="_blank"
+        >
+          <img :src="require('~/assets/svg/github.svg')" alt="Hawk repositories list on github.com">
+          <h4>
+            31 repo
```

Comment:
```suggestion
            32 repo
```

![image](https://user-images.githubusercontent.com/15259299/142010997-81eeb577-34eb-47da-80d7-2c71dba9ec80.png)

Reply:
32 - 1 of old hawk = 31